### DISCUSSION TITLE
How to verify successful setup.

### DISCUSSION THREAD

**Discussion by subdavis** [tomMoulard/fail2ban#27] [Q&A] [answered]
The Traefik Pilot dashboard doesn't even provide validation that I've installed this plugin.  I don't know if I've configured it correctly, or if it's even being used.  I don't know if plugins behave like normal middlewares where I have to enable them on each individual service (or entrypoint) because the docs don't say I have to do this.

I like this idea, but I have very little confidence from the feedback traefik gives that I've set this up correctly.

Could you provide some steps to verify successful installtion?

**Comment by cledavid** [comment, marked as answer]
Hi,

Plugins are still in beta so on the pilot dashboard we still don't see the middleware like fail2ban. 
So far so good it's normal.

We were working on logs about the configuration of fail2ban, a new release is now available : v0.5.0 ([release](https://github.com/tomMoulard/fail2ban/releases/tag/v0.5.0))
We just closed the PR adding more logs.

If you have any other question, don't hesitate

**Reply by subdavis** [reply]
Thanks for the response.  I'll try the new version.

**Reply by subdavis** [reply]
I've upgraded to v0.5.0 and things look to be working well!

## My configuration

Traefik Static:  https://github.com/subdavis/selfhosted/blob/master/docker-compose.yml#L524-L526
Traefik Dynamic: https://github.com/subdavis/selfhosted/blob/master/etc/traefik/rules-fail2ban.yml

Note that it seems obvious, but nothing happens until you actually _use_ the middleware on a router or entrypoint.

i.e. `--entrypoints.websecure.http.middlewares=fail2ban-mddl@file`
